### PR TITLE
update zh-trabns content/zh/docs/concepts/services-networking/ingress…

### PR DESCRIPTION
**Pull request by tanjunchen**
update zh-trans content/zh/docs/concepts/services-networking/ingress-controllers.md

### CODE REVIEW

File: content/zh/docs/concepts/services-networking/ingress-controllers.md [hunk 1]
```diff
@@ -1,116 +1,133 @@
...
+{{< /note >}}
+
+{{% /capture %}}
+
+{{% capture whatsnext %}}
+<!--
+* Learn more about [Ingress](/docs/concepts/services-networking/ingress/).
+* [Set up Ingress on Minikube with the NGINX Controller](/docs/tasks/access-application-cluster/ingress-minikube).
+-->
+* 了解更多关于 [Ingress](/docs/concepts/services-networking/ingress/)。
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/ingress-controllers.md]:
```suggestion
* 进一步了解 [Ingress](/docs/concepts/services-networking/ingress/)。

```

**Comment by xichengliudui** [comment]
/approve
@tanjunchen thanks!

**Pull request merged**